First task completed CMC

c3m3z4c4 opened the pull request:
This is the first task that i've made for this magazine.

c3m3z4c4 committed:
initial commit

c3m3z4c4 committed:
Semantic version
This is a semantic version containind all the cover into a div, after that there are two main blocks.

c3m3z4c4 commented:
This is the second task completed.

crisostomoglobant commented on index.html:
```diff
@@ -5,22 +5,35 @@
+          <h1 class="title">:Into the wild:</h1>
+        </header>
+        <aside class="main-topic-container">
+          <h3 class="main-topic">Looking for exciting experiences?</h3>
```
este h3 debe ser un h2 porque no hay en todo el documento. y no es buena idea saltarte la numeracion desde tu h1

los elementos heading no representan el tamano si no mas bien la importancia del contenido

c3m3z4c4 replied:
Cambios realizados

crisostomoglobant commented on index.html:
```diff
@@ -5,22 +5,35 @@
+        <aside class="main-topic-container">
+          <h3 class="main-topic">Looking for exciting experiences?</h3>
+        </aside>
+        <section class="content-container">
```
`class="content-container"` el nombre de la clase es muy ambiguo. sugiero asignar un nombre que represente mas al contenido
lo mismo para sus clases hijas 

`class="content"`
`".content__pair-option"`

crisostomoglobant commented on index.html:
```diff
@@ -5,22 +5,35 @@
+            <li>Camping Beginers</li>
+            <li class=".content__pair-option">Wild playlist</li>
+            <li>Another challenge?</li>
+            <li class=".content__pair-option">Camping Do's and Don'ts</li>
```
`".content__pair-option"` parece que hay un typo aqui con el punto (.) inicial

c3m3z4c4 replied:
Se elimino esa clase debido a que no se uso

crisostomoglobant commented on index.html:
```diff
@@ -5,22 +5,35 @@
+          </ul>
+        </section>
+        <footer class="pub-date-container">
+          <p class="pub-date">July. 2021</p>
```
dificilmente el elemento Paragraph puede tener contenido dividido para este contenido como la fecha y autor 

sin duda el elemento p es para parrafos 

si tu intencion solo es dar el salto que genera nativamente el Paragraph 
puedes hacerlo con spans y estilos

c3m3z4c4 replied:
Cambios realizados por span

crisostomoglobant commented on styles.css:
```diff
@@ -0,0 +1,110 @@
+  margin: 0;
+  padding: 0;
+  font-size: 62.5%;
+}
```
es muy raro el caso donde un font size este con porcentaje 
no estoy diciendo que este mal 
solo que se puede debatir (en un mundo ideal con tu equipo de trabajo en el proyecto) si efectivamente esta correcto o mejor dicho si es necesario

c3m3z4c4 replied:
Esto lo hice debido a que con ese porcentaje, un REM es equivalente a 10 px cuando se define en otros lados.

crisostomoglobant commented on styles.css:
```diff
@@ -0,0 +1,110 @@
+}
+
+.magazine-container {
+  margin-top: 1%;
```
mismo caso para los paddings y margins 
dificilmente se asigna el valor con porcentaje

c3m3z4c4 replied:
Cambio por REM

crisostomoglobant commented on styles.css:
```diff
@@ -0,0 +1,110 @@
+}
+
+.cover-container {
+  background: url(https://images.unsplash.com/photo-1572304290488-9ea38ef644a9?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=634&q=80)
```
por tema de accesibilidad no es buena idea pintar una imagen de esta manera 
es necesario renderizar/usar el elemento `<img />` con su respectiva descripcion en la propiedad/atributo `alt `
dado que es una imagen de portada la imagen es super importante 

en algunos casos se usa una imagen para acompaniar al contenido 
pero esta se puede ocultar para accesibilidad con la prop `arria-hidden="true"`

more info aca -> https://www.w3.org/WAI/tutorials/images/

c3m3z4c4 replied:
Cambie esto por una imagen con alt.

crisostomoglobant commented on styles.css:
```diff
@@ -0,0 +1,110 @@
+
+.title-container .title {
+  color: var(--gris);
+  font-size: 7rem;
```
great !!
siempre es buena idea manejar los font size con REM 
more info aca ---> 

https://stackoverflow.com/questions/30374863/why-use-rem-instead-px-when-its-the-same-anyway
![image](https://user-images.githubusercontent.com/81988661/121558251-1c045980-c9db-11eb-9df9-717d0664ecb2.png)

c3m3z4c4 replied:
Gracias!

c3m3z4c4 committed:
Changes submitted
[UPDATE] Se realizaron los cambios sugeridos y se elimino codigo comentado y ya no utilizado, el porcentaje en el font-size lo use debido a que con esto, puedo utilizar el REM mas facil, con este porcentage, 1 REM es equivalente a 10 px.

c3m3z4c4 commented:
Ya hice los cambios. excepto el del porcentaje del font-size, ese lo use para poder calcular el tama;o del REM ( con ese porcentaje, 1 REM es equivalente a 10 px)

Saludos.
